app/helpers/config/aca_helper.rb

```diff
@@ -290,10 +290,6 @@ def site_broker_claim_quoting_enabled?
       Settings.site.broker_claim_quoting_enabled
     end
 
-    def calendar_is_enabled?
```

Comment:
Can we remove calendar_enabled setting as well...

`./config/client_config/me/config/settings.yml:96:  calendar_enabled: true
./config/client_config/dc/config/settings.yml:93:  calendar_enabled: true
./config/settings.yml:73:  calendar_enabled: true
./components/notifier/config/settings.yml:93:  calendar_enabled: true
./components/notifier/spec/dummy/config/settings.yml:96:  calendar_enabled: true
./components/transport_profiles/config/settings.yml:93:  calendar_enabled: true
./components/transport_profiles/spec/dummy/config/settings.yml:95:  calendar_enabled: true
./components/ui_helpers/config/settings.yml:93:  calendar_enabled: true
./components/ui_helpers/spec/dummy/config/settings.yml:96:  calendar_enabled: true
./components/transport_gateway/config/settings.yml:93:  calendar_enabled: true
./components/transport_gateway/spec/dummy/config/settings.yml:96:  calendar_enabled: true
./components/benefit_markets/config/settings.yml:93:  calendar_enabled: true
./components/benefit_markets/spec/dummy/config/settings.yml:96:  calendar_enabled: true
./components/benefit_sponsors/config/settings.yml:93:  calendar_enabled: true
./components/benefit_sponsors/spec/dummy/config/settings.yml:96:  calendar_enabled: true
./components/financial_assistance/config/settings.yml:93:  calendar_enabled: true
./components/sponsored_benefits/config/settings.yml:93:  calendar_enabled: true
./components/sponsored_benefits/spec/dummy/config/settings.yml:96:  calendar_enabled: true`

Reply:
I did not know that we have a legacy setting. Good Catch @raghuramg. I will work on it.